src/jit/simd.cpp

```diff
@@ -225,7 +223,7 @@ var_types Compiler::getBaseTypeAndSizeOfSIMDType(CORINFO_CLASS_HANDLE typeHnd,
         // Obtain base type by parsing fully qualified class name.
         //
         // TODO-Throughput: implement product shipping solution to query base type.
-        WCHAR className[256] = {'\\0'};
+        WCHAR className[256];
         WCHAR *pbuf = &className[0];
         int len = sizeof(className)/sizeof(className[0]);
         info.compCompHnd->appendClassName(&pbuf, &len, typeHnd, TRUE, FALSE, FALSE);   
```

Comment:
I did follow this in the appendClassName (the function is below). It is OK to leave the initialization out. The called method does wcscpy_s, so it doesn't matter what the incoming content is. The incoming *pnBufLen is about 128.

int CEEInfo::appendClassName(__deref_inout_ecount(*pnBufLen) WCHAR** ppBuf,
                             int* pnBufLen,
                             CORINFO_CLASS_HANDLE    clsHnd,
                             BOOL fNamespace,
                             BOOL fFullInst,
                             BOOL fAssembly)
{
    CONTRACTL {
        SO_TOLERANT;
        MODE_PREEMPTIVE;
        THROWS;
        GC_TRIGGERS;
    } CONTRACTL_END;

    int nLen = 0;

    JIT_TO_EE_TRANSITION();

    TypeHandle th(clsHnd);
    StackSString ss;
    TypeString::AppendType(ss,th,
                           (fNamespace ? TypeString::FormatNamespace : 0) |
                           (fFullInst ? TypeString::FormatFullInst : 0) |
                           (fAssembly ? TypeString::FormatAssembly : 0));
    const WCHAR* szString = ss.GetUnicode();
    nLen = (int)wcslen(szString);
    if (*pnBufLen > 0)
    {
    wcscpy_s(*ppBuf, *pnBufLen, szString );
    (*ppBuf)[(*pnBufLen) - 1] = W('\0');
    (*ppBuf) += nLen;
    (*pnBufLen) -= nLen;
    }

    EE_TO_JIT_TRANSITION();

    return nLen;
}